morpcc/navigator.py

```diff
@@ -65,14 +65,14 @@ def __init__(self, schema, entity, request):
 
     def add_attribute(
         self,
-        name,
-        type_,
-        title,
-        description=None,
-        required=False,
-        primary_key=False,
-        dictionaryelement=None,
-        allow_invalid=False,
+        name: str,
+        type_: str,
+        title: str,
+        description: typing.Optional[str] = None,
+        required: bool = False,
+        primary_key: bool = False,
+        dictionaryelement: typing.Optional[str] = None,
```

Comment:
nope .. 

read ``create_app.py`` and ``navigator.py``
-  ``nav`` variable stores ``Navigator`` 
- ``dd`` variable get from ``nav.datadictionary``, which if your read ``navigator.py``, ``datadictionary`` attribute of ``Navigator`` is a ``DataDictionaryBrowser`` object.
- ``dd['cifid']`` will call ``DataDictionaryBrowser.__getitem__('cifid')``, which returns ``DictionaryEntityNavigator``

so whats type type of ``dictionaryelement`` ?